services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None) -> QuerySet:
+    movies_queryset = Movie.objects.all()
+    if genres_ids:
+        movies_queryset = movies_queryset.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        movies_queryset = movies_queryset.filter(actors__id__in=actors_ids)
+
+    return movies_queryset
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: list[int] = None,
+                 actors_ids: list[int] = None) -> Movie:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]